### PR TITLE
Edit UML diagram

### PR DESCRIPTION
**Pull request by teamotitan**
Previously, UML diagram was not tailored towards our application

Let's 
*edit UML diagram such that it is tailored to our application

### CODE REVIEW

**Review by Feng1231 — APPROVED** [review]
LGTM!

File: docs/diagrams/BetterModelClassDiagram.puml [hunk 1]
```diff
@@ -4,18 +4,18 @@ skinparam arrowThickness 1.1
 skinparam arrowColor MODEL_COLOR
 skinparam classBackgroundColor MODEL_COLOR
 
-AddressBook *-right-> "1" UniquePersonList
+AddressBook *-right-> "1" UniqueStudentList
```
Comment (TY1Fan) [inline docs/diagrams/BetterModelClassDiagram.puml:7]: Nice catch! Great that you updated the Person to Student.

**Review by TY1Fan — APPROVED** [review]
LGTM

**Review by DennieDan — APPROVED** [review]
LGTM

**Pull request merged**